backend/src/utils/interfaces.ts

```diff
@@ -0,0 +1,25 @@
+//TODO find a way to enforce regex format for email inside an interface.
+export interface Company extends Object {
```

Comment:
pretty sure i have talked about this before, but i really have problems understanding how/why a company should be a user. 
Who should have access to this user and what should this user be able to do that a company user should not?

Reply:
It was mostly to control the users, and their permission, but i just realised what i think you've been trying to convey the whole time... 😅 

I'll remove the company as a user from the db and their role creation from the route. 

I'll still neeed the company interface, that interface is only used to ensure that when a new company is created that we know what type the dataobjects has.

Reply:
yes, nice. Then we are on the same page :+1: